mne/viz/backends/_utils.py

```diff
@@ -160,10 +162,13 @@ def _init_mne_qtapp(enable_icon=True, pg_app=False, splash=False):
         app = QApplication([app_name])
     app.setApplicationName(app_name)
     app.setOrganizationName(organization_name)
-    try:
-        app.setAttribute(Qt.AA_UseHighDpiPixmaps)  # works on PyQt5 and PySide2
-    except AttributeError:
-        pass  # not required on PyQt6 and PySide6 anyway
+    qt_version = _check_qt_version(check_usable_display=False)
+    # high DPI is enabled by default in PyQt6
+    if _compare_version(qt_version, '<', '6.0'):
+        try:
```

Comment:
I don't think that the `try` - `except` is needed anymore, but I wanted to be conservative in this change. Let me know if you think that I should remove it.

Reply:
Agreed you can remove it